views/basket/mobile.php

```diff
@@ -0,0 +1,354 @@
+<div class="visible-xs visible-sm">
+    <h1>Your Basket</h1>
+    <?php
+
+    foreach ($items as $item) { ?>
+
+        <div class="row bordered-row">
+
+            <?php
+
+            if (!empty($item->variant->featured_img)) {
+
+                $featuredImg = $item->variant->featured_img;
+
+            } elseif (!empty($item->product->featured_img)) {
+
+                $featuredImg = $item->product->featured_img;
+
+            } else {
+
+                $featuredImg = false;
+            }
+
+
+            if ($featuredImg) {
+
+                echo '<div class="col-xs-3">';
+
+                    $url = cdn_thumb($featuredImg, 175, 175);
+                    echo img(array('src' => $url, 'class' => 'img-thumbnail'));
+
+                echo '</div>';
+
+            } else {
+
+                echo '<div class="col-xs-12">';
+
+            }
+
+            ?>
+
+            <div class="col-xs-7">
+
+                <?php
+
+                // --------------------------------------------------------------------------
+
+                //  Label
+                echo anchor($item->product->url, '<strong>' . $item->product->label . '</strong>');
+
+                if ($item->variant->label !== $item->product->label) {
+
+                    echo '<br />';
+                    echo '<em>' . $item->variant->label . '</em>';
+                }
+
+                // --------------------------------------------------------------------------
+
+                //  To order?
+                if ($item->variant->stock_status == 'TO_ORDER') {
+
+                    echo '<p class="text-muted">';
+                        echo '<small>';
+                            echo '<em>Lead time: ' . $item->variant->lead_time . '</em>';
+                        echo '</small>';
+                    echo '</p>';
+                }
+
+                // --------------------------------------------------------------------------
+
+                //  Collection Only
+                if ($item->variant->shipping->collection_only) {
+
+                    echo '<div class="alert alert-warning alert-mini">';
+                        echo '<strong>Note:</strong> Collection only.';
+                    echo '</div>';
+                }
+     
+                ?>
+
+            <div>
```

Comment:
Line 81 is a div that is just used for creating a block and new line. I close this on line 124. (fixed indentation now)

Line 126 is the closing of `col-xs-7` on line 42